src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,53 @@
+.clock {
+  height: 300px;
+  width: 300px;
+  border-radius: 50%;
+  border: 2px solid black;
+  margin: 0 auto;
+  transform: translateY(50%);
+  background-color: #ff0;
+
+  &__seconds {
+    height: 100px;
+    width: 2px;
+    border-radius: 50%;
+    background-color: #f00;
+    animation: move 60s steps(60) infinite;
+  }
+
+  &__hours {
+    height: 140px;
+    width: 4px;
+    border-radius: 50%;
+    background-color: #00f;
+    animation: move 3600s linear infinite;
+  }
+
+  &__center {
+    z-index: 1;
+    position: absolute;
+    height: 15px;
+    width: 15px;
+    border-radius: 50%;
+    background-color: #000;
+    right: 48%;
+    top: 48%;
```

Comment:
it's better to set 50% and then translate the block like `transform: translate(-50%, -50%)`